src/components/Layout/Header/SideNavContent.tsx

```diff
@@ -3,8 +3,6 @@ import type { FlexProps } from '@chakra-ui/react'
 import { Box, Flex, IconButton, Stack, useMediaQuery } from '@chakra-ui/react'
 import { lazy, memo, Suspense, useCallback, useMemo } from 'react'
 import { useTranslate } from 'react-polyglot'
-import { useHistory } from 'react-router'
-import { ArkeoIcon } from 'components/Icons/Arkeo'
```

Comment:
I think we can also remove it from the project

Reply:
the icon? yeah totally. I left some artifacts but maybe it's better to remove all.